Cargo.lock

```diff
@@ -54,9 +54,9 @@ dependencies = [
 
 [[package]]
 name = "aes"
-version = "0.8.3"
+version = "0.8.4"
```

Comment:
is it usual to update everything?

Reply:
Sometimes it does, I basically re-build the `Cargo.lock` file from scratch after merging master.